src/applications/vaos/express-care/redux/actions.js

```diff
@@ -103,12 +103,9 @@ export function fetchRequestLimits() {
         moment(),
       ).map(win => win.facilityId);
 
-      // Temporarily limit concurrent calls to 5 while we
-      // wait for a new endpoint that will accept multiple facilityIds
-      const requestLimits = await Promise.all(
-        activeFacilityIds
-          .slice(0, 5)
-          .map(facilityId => getRequestLimits(facilityId, EXPRESS_CARE)),
+      const requestLimits = await getRequestLimits(
+        activeFacilityIds.slice(0, 5),
```

Comment:
I don't think you need to limit this to 5 here, I think you pull all of the facility ids we have. We were limiting it to 5 to not make a bunch of api requests at once and overwhelm the user's browser or the backend, but now they're all in one api request.